### PR TITLE
API1600 Support for Storage Pool

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[API1600 Support for Storage Pool]

### Issues Resolved
[]

### Check List
- [x] New functionality includes testing.
  - [x]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,11 +3,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 # [v1.3.0](unreleased)
 ### Notes
-- This release supports for API 800,1000,1200  for resource listed below.
+- This release supports for API 800,1000,1200,1600  for resource listed below.
 
 ### Oneview Features supported
 - Hypervisor Manager
```
Comment (AnushaY1916) [inline CHANGELOG.md:9]: Hypervisor manager and server certificate resources are not yet supported for API 1600 version, remove these entries as well.

File: oneview/resource_storage_pool.go [hunk 2]
```diff
@@ -145,16 +159,19 @@ func resourceStoragePoolRead(d *schema.ResourceData, meta interface{}) error {
 	rawdevspecificattributes := storagePool.DeviceSpecificAttributes
 	devspecificattributes := make([]map[string]interface{}, 0)
 	devspecificattributes = append(devspecificattributes, map[string]interface{}{
-		"device_id":            rawdevspecificattributes.DeviceID,
-		"capacity_limit":       rawdevspecificattributes.CapacityLimit,
-		"device_speed":         rawdevspecificattributes.DeviceSpeed,
-		"domain":               rawdevspecificattributes.Domain,
-		"supported_raid_level": rawdevspecificattributes.SupportedRaidLevel,
-		"uuid":                 rawdevspecificattributes.Uuid,
+		"device_id":                rawdevspecificattributes.DeviceID,
```
Comment (AnushaY1916) [inline oneview/resource_storage_pool.go:162]: I think these are lines that are giving gofmt errors.

Comment (nabhajit-ray) [reply oneview/resource_storage_pool.go]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (nabhajit-ray) [reply CHANGELOG.md]: Done

**Pull request merged**